app/views/candidate_mailer/_get_support.text.erb

```diff
@@ -1,3 +1,7 @@
 # Get support
 
-Contact [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk) if you have problems applying online or want to give feedback.
+You can chat to a Get Into Teaching adviser online for help and advice:
+
+[https://beta-getintoteaching.education.gov.uk/#talk-to-us](https://beta-getintoteaching.education.gov.uk/#talk-to-us)
```

Comment:
Sorry i hadn't picked this up by looking at the ticket before, but does it seem right that we are sending everyone to GIT rather than giving linking them to our services support? Let's check on Monday.